conda/py312.yaml

```diff
@@ -0,0 +1,14 @@
+zip_keys:
+  - [python, numpy, python_impl, is_python_min]
+
+python:
+  - 3.12.* *_cpython
+
+numpy:
+  - 1.26
+
+python_impl:
+  - cpython
+
+is_python_min:
+  - false
```

Comment:
blank end of line missing

Reply:
Okay ! In fact I do not use this file anymore, but use --python instead. The file (and the missing blank line will probably go away!)

Reply:
The file has been removed